packages/eventwebhook/src/eventwebhook.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const ecdsa = require('@starkbank/ecdsa');
+const Ecdsa = ecdsa.Ecdsa;
+const Signature = ecdsa.Signature;
+const PublicKey = ecdsa.PublicKey;
+
+class EventWebhook {
+  /**
+   *
```

Comment:
Missing summaries.

Reply:
Adding now. Should I submit a separate PR?

Reply:
done.